### PR TITLE
updating

### PR DESCRIPTION
**Pull request by michaelca2024**
Done
#8 

To do

### CODE REVIEW

File: _slides/04-michaelca2024.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello Michael here
```
Comment (michaelca2024) [inline _slides/04-michaelca2024.md:6]: This is good

Comment (jharstad) [inline _slides/04-michaelca2024.md:6]: Looks good

**Review by jharstad — APPROVED** [review]
approved

File: _slides/04-michaelca2024.md (hunk 1 above)
Comment (Divshub13) [inline _slides/04-michaelca2024.md:6]: Looks good

**Review by Divshub13 — APPROVED** [review]

**Pull request merged**